### ISSUE TITLE
Question related to drillDown method: undefined is not iterable error

### ISSUE THREAD

**Issue by DPHonys** [cube-js/cube#6913]
**Problem**
Hi, I seem to have some difficulty with drillDown method on resultSet in my vue project...

If I try to create a drillDown query with resultSet.drillDown method and the main query has an undefined date range (aka All time) but set granularity (e.g. month) this error `Error in v-on handler: "TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))` occures. But if I set the dateRange to for example "last year" the drillDown method works correctly and I receive the correct data.

Used packages are @cubejs-client/core and @cubejs-client/vue both version [0.33.12](https://www.npmjs.com/package/@cubejs-client/core/v/0.33.12).

I even tried to create a node project (@cubejs-client/core v0.33.12) to test it but it has the same problem (if dateRange is not set and granularity is it fails with mentioned error).. but if dateRange is also set it works flawlessly.
```javascript
const cubejs = require('@cubejs-client/core')

const cubeAPI = new cubejs.CubejsApi("", {
    apiUrl: `http://localhost:4000/cubejs-api/v1`,
})

const query = {
    "measures": [
      "TaskView.count"
    ],
    "timeDimensions": [
      {
        "dimension": "TaskView.createdAt",
        "granularity": "month"
      }
    ],
    "order": {
      "TaskView.count": "desc"
    },
    "dimensions": [
      "TaskView.type"
    ]
  };

const drillDownLocator = {
  "xValues": [
      "STOCK_PICKING",
      "2021-12-01T00:00:00.000"
  ],
  "yValues": [
      "TaskView.count"
  ]
}

const pivotConfig = {
  "fillMissingDates": false,
  "joinDateRange": false,
  "x": [
      "TaskView.type",
      "TaskView.createdAt.month"
  ],
  "y": [
      "measures"
  ]
}

cubeAPI.load(query).then((resultSet) => {
    //console.log(JSON.stringify(resultSet, null, 2));
    const drillDownQuery = resultSet.drillDown(drillDownLocator, pivotConfig);
    console.log(JSON.stringify(drillDownQuery, null, 2));
    return cubeAPI.load(drillDownQuery);
}).then((resultSet) => {
    //console.log(JSON.stringify(resultSet, null, 2));
})
```
(pivotConfig and drillDownLocator are taken from the vue project)

So I did some digging in the debugger and it looks like this issue is connected to the commit in this pull request [#6639](https://github.com/cube-js/cube/pull/6639) because the drillDown method fails on line [173](https://github.com/cube-js/cube/blob/879c2f9e16a3237e7a8d2ddedb470c0a1ec5645c/packages/cubejs-client-core/src/ResultSet.js#L173) `const [originalStart, originalEnd] = originalTimeDimension.dateRange;` where the if on previous line checks for originalTimeDimension but it does not check if it contains dateRange so it fails and produces error I've mentioned before.

I tried downgrading to a package version where the mentioned pull request is not merged and the drillDown method works correctly for both Vue and test node project - @cubejs-client/core@[0.33.0](https://www.npmjs.com/package/@cubejs-client/core/v/0.33.0). So it seems like the mentioned pull request might be the problem.

It might be a bug caused by the pull request but I might be missing something in the 0.33.12 version... Thanks for reading will be happy with any help 🙂

**Comment by DPHonys** [comment]
Fixed in [v0.33.58](https://github.com/cube-js/cube/releases/tag/v0.33.58). 🐣

**Comment by DPHonys** [comment]
Fixed in [v0.33.58](https://github.com/cube-js/cube/releases/tag/v0.33.58). 🐣
